### PR TITLE
Readme facelift

### PR DESCRIPTION
**Pull request by jayrod**

### CODE REVIEW

File: readme_files/shout_out.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[CephFS Shell](http://docs.ceph.com/docs/master/cephfs/cephfs-shell/),'[Ceph](https://ceph.com/) is a distributed object, block, and file storage platform'
+[JSShell](https://github.com/Den1al/JSShell),An interactive multi-user web JavaScript shell.
+[psiTurk](https://psiturk.org),An open platform for science on Amazon Mechanical Turk
+[Poseidon](https://github.com/CyberReboot/poseidon),Leverages software-defined networks (SDNs) to acquire and then feed network traffic to a number of machine learning techniques.
+[Unipacker](https://github.com/unipacker/unipacker),Automatic and platform-independent unpacker for Windows binaries based on emulation
+[FLASHMINGO](https://github.com/fireeye/flashmingo),Automatic analysis of SWF files based on some heuristics. Extensible via plugins.
+[tomcatmanager](https://github.com/tomcatmanager/tomcatmanager),A command line tool and python library for managing a tomcat server
+[Expliot](https://gitlab.com/expliot_framework/expliot),Internet of Things (IoT) exploitation framework
+[mptcpanalyzer](https://github.com/teto/mptcpanalyzer),Tool to help analyze mptcp pcaps
```
Comment (anselor) [inline readme_files/shout_out.csv:11]: This needs to be changed to: https://github.com/teto/pymptcpanalyzer

File: readme_files/shout_out.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Application Name, Description
+[Jok3r](http://www.jok3r-framework.com),Network & Web Pentest Automation Framework
+[CephFS Shell](http://docs.ceph.com/docs/master/cephfs/cephfs-shell/),'[Ceph](https://ceph.com/) is a distributed object, block, and file storage platform'
```
Comment (anselor) [inline readme_files/shout_out.csv:3]: This link doesn't appear to go anywhere.

File: readme_files/shout_out.csv [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+[JSShell](https://github.com/Den1al/JSShell),An interactive multi-user web JavaScript shell.
```
Comment (anselor) [inline readme_files/shout_out.csv:4]: This project has had no activity for 3 years. Possibly defunct?

File: readme_files/shout_out.csv [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+[psiTurk](https://psiturk.org),An open platform for science on Amazon Mechanical Turk
+[Poseidon](https://github.com/CyberReboot/poseidon),Leverages software-defined networks (SDNs) to acquire and then feed network traffic to a number of machine learning techniques.
+[Unipacker](https://github.com/unipacker/unipacker),Automatic and platform-independent unpacker for Windows binaries based on emulation
+[FLASHMINGO](https://github.com/fireeye/flashmingo),Automatic analysis of SWF files based on some heuristics. Extensible via plugins.
```
[the remark this answers is not in the document]
Comment (jayrod) [reply readme_files/shout_out.csv:8]: I hesitate to remove them as they served as great examples for me when I first started and was looking for how complex applications actually structured cmd2 projects.

File: README.md [hunk 5]
```diff
@@ -1,62 +1,64 @@
...
+The developers toolbox
+----------------------
+
+![system schema](https://github.com/python-cmd2/cmd2/blob/README_facelift/.github/images/graph.drawio.png)
 
-Main Features
+
+When creating solutions developers have no shortage of tools in the belt when it comes to rich and smart user interfaces. `cmd2` deftly fills the niche between high [ease of command discovery](https://clig.dev/#ease-of-discovery) and smart workflow automation systems. The above graph is over simplified helps to illustrate when one should reach for cmd2 as the right choice. CLIs have long been the go to pick for those concerned with automation. System administrators have long been duct taping together precarious workflows based on a menagerie of simple command line tools created by strangers on github and the guy down the hall. When CLIs become significantly complex the ease of discoverability tends to fade quickly. Web and GUIs on the other hand are first in class when allowing users to easily discover functionality. The price we pay for beautifully colored displays is it is exceedingly difficult to aggregate many of these applications into larger systems.
```
Comment (tleonhardt) [inline README.md:34]: Need to slightly reword `The above graph is over simplified helps to illustrate ...`

File: README.md [hunk 6]
```diff
@@ -220,121 +94,6 @@ Tutorials
     * Basic cookiecutter template for cmd2 application : https://github.com/jayrod/cookiecutter-python-cmd2
     * Advanced cookiecutter template with external plugin support : https://github.com/jayrod/cookiecutter-python-cmd2-ext-plug
 
-Example Application
```
Comment (tleonhardt) [inline README.md:223]: I like showing at least a basic example application in the README.  I'm game for simplifying it.  But I want someone to quickly see what a "Hello World" `cmd2` application might look like on the "front page".

File: README.md [hunk 7]
```diff
@@ -348,30 +107,35 @@ If you think you've found a bug, please first read through the open [Issues](htt
...
-    * A command line tool and python library for managing a tomcat server
-* [Expliot](https://gitlab.com/expliot_framework/expliot)
-    * Internet of Things (IoT) exploitation framework
-* [mptcpanalyzer](https://github.com/teto/mptcpanalyzer)
-    * Tool to help analyze mptcp pcaps
-* [clanvas](https://github.com/marklalor/clanvas)
-    * Command-line client for Canvas by Instructure
+* [FLASHMINGO](https://github.com/fireeye/flashmingo) 
+
+Version 2.0 Notes
```
Comment (tleonhardt) [inline README.md:131]: We can probably just delete these "Version 2.0 Notes"

**Review by tleonhardt — CHANGES_REQUESTED** [review]
Overall it looks like a great facelist!  Thanks ;-)

File: README.md (hunk 5 above)
Comment (jayrod) [reply README.md:34]: ya I tend to babble. I'm open to suggestions.

**Comment by anselor** [comment]
@jayrod There are quite a lot of incremental changes here. Unless there are significant points that we'd want to keep for historical purposes I'd suggest doing a rebase and squash of your changes.

**Comment by jayrod** [comment]
I'm not sure if I squashed all of the comments or not. I feel like I did.. but not sure how to check it.

**Comment by jayrod** [comment]
I feel like I'm going to have to manually just delete this branch and start over.

**Comment by anselor** [comment]
@jayrod https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec

I can try to talk you through it later but that article seems to cover things in detail.

**Comment by tleonhardt** [comment]
I did a squash and merge from Git.  It solved the problems I think

**Comment by jayrod** [comment]
Appreciate it. Before I start a new branch I'll practice it. I think I was running random rebase commands and pulling stuff from master.

**Comment by anselor** [comment]
Read through the page I linked. If you have questions I'd be happy to answer them.

**Pull request merged**